android-lib/src/main/java/com/kinvey/android/store/UserStore.java

```diff
@@ -422,6 +422,13 @@ public void keepOfflineStorageOnLogout(){
     }
 
     /**
+     * Get a flag which allow local offline storage to persist after calls to logout.
+     */
+    public boolean isKeepOfflineStorageOnLogout() {
```

Comment:
@misha-vakulich How does this `clearStorage` flag set?  I see that there is a default value of `true`, and a method to set it explicitly to `false`, but I don't see a way of toggling this value.